content/en/blog/falco-0-30-0.md

```diff
@@ -0,0 +1,90 @@
+
+```yaml
+metadata_download:
+  max_mb: 100
+  chunk_wait_us: 1000
+  watch_freq_sec: 1
+```
+
+This release also adds the ability to [export](https://github.com/falcosecurity/falco/pull/1714) rule tags and event source in gRPC and JSON outputs! This behavior can be [configured](https://github.com/falcosecurity/falco/pull/1733), and enables Falco event consumers, such as Falco Sidekick, to take full advantage of Falco's event tagging feature. Happy tagging :)
+
+### Libs plugin system
+
+With Falco 0.30.0, a [proposal](https://github.com/falcosecurity/falco/blob/master/proposals/20210501-plugin-system.md) for a libs [plugin system](https://github.com/falcosecurity/plugins) has been accepted, and we couldn't be more excited! The possibilities are limitless! 🎉 
+
+Plugins will allow users to easily extend the functionality of the libraries and, as a consequence, of Falco and any other tool based on the libraries. This proposal, in particular, focuses on two types of plugins: source plugins and extractor plugins. A _source_ plugin implements a new sinsp/scap event source (e.g., "k8s\_audit"), while an _extractor_ plugin focuses on field extraction from events generated by other plugins, or by the core libraries.
+
+Plugins are dynamic libraries (.so files in Unix, .dll files in windows) that export a minimum set of functions that the libraries will recognize. They can be written in any language, as long as they export the required functions. Go, however, is the preferred language to write plugins, followed by C/C++. To facilitate the development of plugins, a [golang SDK](https://github.com/falcosecurity/plugin-sdk-go) has been developed. 
+
+Both the [plugin system](https://github.com/falcosecurity/plugins) and [SDK](https://github.com/falcosecurity/plugin-sdk-go) are now incubating projects in the Falco organization, and include a set of initial [examples](https://github.com/falcosecurity/falco/blob/master/proposals/20210501-plugin-system.md#examples). We invite the community to try them out, contribute new plugins, and join efforts to build together the foundation for cloud-native runtime security! 🚀
+
+### New Falco release schedule
+
+Finally, after discussing with the community, a new [release schedule](https://github.com/falcosecurity/falco/pull/1711) has been approved for Falco. New releases are now due to happen _three_ times per year: at the end of _January_, _May_, and _September_. We will continue to release hot fixes and minor patches in between major releases. As always, feedback, bug reports, and contributions are welcome! :)
+
+---
+
+## Try it!
+
+As usual, in case you just want to try out the stable Falco 0.29.0, you can install its packages following the process outlined in the docs:
```

Comment:
```suggestion
As usual, in case you just want to try out the stable Falco 0.30.0, you can install its packages following the process outlined in the docs:
```

Reply:
Thanks @jasondellaluce! I fixed the release number.